### ISSUE TITLE
[20210328] Weekly Arxiv 만담 (22일은 개인 사정으로 쉽니다 ㅎㅎ)

### ISSUE THREAD

**Issue by jungwoo-ha** [jungwoo-ha/WeeklyArxivTalk#3]
* AI News
  * Conference
    * ICML 2021 rebuttal
   ![image](https://user-images.githubusercontent.com/11782739/112752311-71041680-900d-11eb-9e4a-afd7d4f27d5f.png)
    * ACL 2021 rebuttal
    * Interspeech 2021 deadline
  * https://www.youtube.com/watch?v=iYwlmLLKQc8 (KAIST AI 대학원 서민준 교수님 Trend in Big LM 온라인 톡)
  * https://ai.googleblog.com/2021/03/contactless-sleep-sensing-in-nest-hub.html
    * 비접촉식으로 수면 모니터링
    * https://drive.google.com/file/d/1GcTyfR1s-ob5JtQwFG6bh4EFpd5mCk0i/view 
  * NAVER CLOVA AI Rush 2021: https://campaign.naver.com/clova_airush/
    * 4월 16일까지 지원합니다.
    * 문제는 1라운드 6개 / 2라운드 8개, 총상금 2억, 라운드별 수행비 320만원, 리얼월드 데이터/문제
* Arxiv
  * FastNeRF: High-Fidelity Neural Rendering at 200FPS (https://arxiv.org/abs/2103.10380 )
    * DONeRF 도 15FPS도 대단하다 생각했는데 무려 200FPS
    * NeRF가 구글, 페북만의 전유물이 아니다! MS도 있다.
    * 공간좌표와 각도 좌표를 분리하고 injection하는 스타일로 하고 caching 전략을 사용.
    * in-the-wild에서도 잘되면 좋겠는데..
  * Preregistering NLP research (https://arxiv.org/abs/2103.06944 )
    * 이제 NLP연구에서도 뭐할지 사전등록 하자 (심리학, 의학에선 그리 한다고...)
    * 여러분의 생각은?
  * PlenOctrees for Real-time Rendering of Neural Radiance Fields (https://arxiv.org/abs/2103.14024 )
    * 일단 데모: https://alexyu.net/plenoctrees/
  * Swin Transformer: Hierarchical Vision Transformer using Shifted Windows (https://arxiv.org/abs/2103.14030 )
    * COCO detection: box AP 58.7!
    * ViT 구조에서 hierarchical 하게 feature 쌓아 올리기 + window partition이 shifting --> self-attention이 새로운 윈도우 계산 효과+경계부분 강화
    * 그런데 좀 복잡하긴 하네요. 복잡한 것 대비 throughput (images / s)은 좋은 편
    * 역시 숫자 뽑아내는건 MSRA가 잘함.
  * Hurdles to Progress in Long-form Question Answering (https://arxiv.org/abs/2103.06332 )
    * 단답형 말고 서술형 Open-domain QA
    * https://ai.googleblog.com/2021/03/progress-and-challenges-in-long-form.html?m=1
  * Towards Few-Shot Fact-Checking via Perplexity (https://arxiv.org/abs/2103.09535 )
    * HKUST의 한국인 분들.. (과연 Pascale Fung)
    * PLM의 이점을 fact-checking에서도 살린다.
    * 특히 막강한 few-shot 성능 (example 2개만으로도 기존 SOTA 잡음)
    * evidence-conditioned LM의 PPL을 활용한 few shot (evidence 없는 주장은 GPT2기준 PPL이 높다)
    * COVID-19 연관 팩트 체크 데이터는 덤
  * One-Shot GAN: Learning to Generate Samples from Single Images and Videos (https://arxiv.org/abs/2103.13389  )
  * NAS-BENCH-ASR: REPRODUCIBLE NEURAL ARCHITECTURE SEARCH FOR SPEECH RECOGNITION (https://openreview.net/pdf/77fdff265261021a568029907ab02c0f1f6e4639.pdf )
    *  by U of Cambridge and Samsung AI Center Cambridge (appear at ICLR 2021)
    * 이제는 ASR에서도 NAS-bench 가
  * Adversarial Feature Augmentation and Normalization for Visual Recognition (https://arxiv.org/abs/2103.12171 )
    * Adversarial feature-level data augmentation
    * 여러 Adversarial 을 통해 batch-norm statistics를 계산하고 이것을 clean image에 다시 주입하여 augment 하는 기법.
    * 다양한 태스크에서 조금씩 성능향상이 확인
    * 과거 adv aug (AdvProp) 기법대비 훈련시간이 줄긴해도 Baseline대비 3배정도 늘어난 것에 비하면 margin 인 큰 걸까?
  * Efficient Feature Transformations for Discriminative and Generative Continual Learning (https://arxiv.org/abs/2103.13558v1 )
    * appear at CVPR2021
    * 추가 파라미터를 활용하는 스타일의 CIL 과 task prediction
    * LSUN 데이터로 생성모델에서도 적용.
    * 기존 기법대비 feature transformation을 위해 추가로 들이는 파라미터가 상대적으로 매우 작다고 함.
  * Rethinking Self-Supervised Learning: Small is Beautiful (https://arxiv.org/pdf/2103.13559v1.pdf )
    * 기존 SSL의 대세인 "데이터 많이, 백본 크게, 해상도도 크게" 철학 에 반대하는 실험결과
    * 해상도는 작게 하고, 백본도 일부만 쓰고, 학습하는 데이터의 양도 줄인다. 에폭과 해상도는 함께 고려
    * 타겟 태스크 데이터가 작으면 굳이 큰 백본 없어도 된다.  
    * 자세하게 읽은거 아님 주의 (실험 공정성 등 체크해 봐야함 ㅋㅋ)
    * https://github.com/CupidJay/Scaled-down-self-supervised-learning
  * An Approach to Improve Robustness of NLP Systems against ASR Errors (https://arxiv.org/pdf/2103.13610v1.pdf )
    * 화웨이 Noah's ark lab
    * NLP 인식류 (아마도 NLU 류인듯) 에서 ASR 에러에 robust 하도록 파이프라인
    * Spoken 번역과 SLU에서 결과 보여줌. 기존의 E2E SLU에 비해 좋나? FSC 성능을 봐선 그런것 같지는 않고..
    * 대신 좀저 general하게 적용가능한 프레임웍이라는 장점?
  * SubSpectral Normalization for Neural Audio Data Processing (https://arxiv.org/abs/2103.13620v1 )
    * Qualcomm AI Research
    * input frequency dimension 을 그룹단위로 쪼개서 normalization 
    * ASR 쪽은 아니고 classification 류 (keyword spotting 포함) spectrogram 기반 ASR에도 잘 동작하면 좋겠네요.
  * A Survey of Quantization Methods for Efficient Neural Network Inference (https://arxiv.org/abs/2103.13630v1 )
    * from UC Berkeley (북챕터)
    * 경량화 quantization 실제 적용등에 관심 있는 분들이나 연구하시는 분들에게 많은 도움이 될 듯
  * Group-CAM: Group Score-Weighted Visual Explanations for Deep Convolutional Networks (https://arxiv.org/abs/2103.13859v1 )
    *  뭔가 class activation을 쪼개고 transformation 후에 다시 weighted sum 해서 보정하는 느낌
    * 기존 CAM보단 좀더 정확하게 visualization 되는 것 같긴하나...
    * abstract가 한컬럼을 다 먹는걸 보니.. 연구내용의 contribution과 상관없이 논문으로선 첫인상이 별루...
  * An Image is Worth 16x16 Words, What is a Video Worth? (https://arxiv.org/pdf/2103.13915v1.pdf )
    * 제목에서 알수 있듯이 video action recognition 용 video transformer
    * 기본적으로 clip을 frame 샘플링하고 이미지는 vision transformer 스타일로 쪼개서 입력 --> spatial transformer
    * 그리고 이들을 Temporal transformer 로 연결 (이미지가 문장, 비디오는 paragraph 로 가정해서 시간축으로 붙임)
    * Inference 에서 필요로 하는 frame 수가 작아서 정확도도 높은데 속도도 빠르다고 함.
  * USB: Universal-Scale Object Detection Benchmark (https://arxiv.org/abs/2103.14027v1 )
    * 일단 이름이 어그로를 끌고
    * COCO, Waymo Open dataset, Manga103 을 기반으로 scale-invariant 한 벤치마크와 성능평가 프로토콜 제시
    * 실험이 굉장히 방대하고 protoocol 도 꽤 잘 정리됨.
    * 일본인 연구자인데 이걸 독립연구자 자격으로 모두 수행함.. 대단한 의지...
  * UNICORN on RAINBOW: A Universal Commonsense Reasoning Model on a New Multitask Benchmark (https://arxiv.org/abs/2103.13009v1 )
    *  Commonsense AI 대가인 UW 최예진 교수님 AAAI 2021 논문의 풀버전
    * 새로운 commonsense 벤치마크와 어마무시한 실험 + commonsense AI  모델 + 소스코드 공개되어 있음.
  * Knowledge-aware Contrastive Molecular Graph Learning (https://arxiv.org/pdf/2103.13047v1.pdf )
    * 신약개발 위한 분자구조 예측 연구
    * Domain knowledge를 효과적으로 인코딩에 녹여 넣기 위해 비슷한 화학식이지만 다른 기능을 하는 분자구조들과의 contrastive learning을 하도록 해서 knowledge를 밀어 넣는 것 같음
  * Representing Numbers in NLP: a Survey and a Vision (https://arxiv.org/abs/2103.13136v1 )
    * 제목만 봐도 땡기는 연구. NAACL2021 발표예정
    * 지금까지 숫자 encoding 해왔던 다양한 방법을 몇가지로 구분해서 요약정리. 그리고 앞으로 가야할 방향 논의
  * Dynamic Slimmable Network (https://arxiv.org/abs/2103.13258v1 )
    * Pruning은 실제로는 여러 연산때문에 실효성이 좀 떨어짐
    * one-shot NAS 스타일로 gating + slimmalble 구성으로 dynamic 하게 slimmalble network 활요
    * 실제로 속도가 엄청 빨리진다고... (mobilenet, ResNet 기준)

**Comment by jshin49** [comment]
Open Domain Question Answering over Tables via Dense Retrieval (https://arxiv.org/pdf/2103.12011v1.pdf)
- 기존 Open-domain QA 는 text 만 retrieve 했는데, table 에서 retrieve 할 수 있는 연구
- + NaturalQuestions data 를 Table QA 형식에 맞게 subset 을 변환

**Comment by jshin49** [comment]
Gradient Regularized Contrastive Learning for Continual Domain Adaptation (https://arxiv.org/pdf/2103.12294v1.pdf)
- Continual Learning + Domain Adaptation
- Contrastive Learning + Gradient Regularization
